ocaml/xapi-cli-server/records.ml

```diff
@@ -2628,12 +2628,12 @@ let pool_patch_record rpc session_id patch =
         (fun x -> Client.Host_patch.get_host ~rpc ~session_id ~self:x)
         host_patch_refs
     in
-    let host_uuids =
+    let hosts =
       List.map
```

Comment:
Revert this

Reply:
https://github.com/xapi-project/xen-api/pull/4966/commits/57aefc95bcbb4244cad32e91cefc2c3902675c9c